src/renderer/pages/Project.tsx

```diff
@@ -84,24 +83,21 @@ const ProjectPage = () => {
               {JSON.stringify(currentProject)}
             </pre>
             <p>Buttons to demo undo</p>
-            <StandardButton onClick={deleteEverySecondWord}>
+            <Button onClick={deleteEverySecondWord}>
               Delete Every Second Word
-            </StandardButton>
-            <StandardButton onClick={changeRandomWordToSwamp}>
+            </Button>
+            <Button onClick={changeRandomWordToSwamp}>
               Change a random word to &lsquo;swamp&rsquo;
-            </StandardButton>
-            <StandardButton
-              onClick={dispatchUndo}
-              disabled={undoStack.index <= 0}
-            >
+            </Button>
+            <Button onClick={dispatchUndo} disabled={undoStack.index <= 0}>
```

Comment:
Again not a blocker, but if you have a minute can you remove the undo/redo buttons please as their functionality exists in the menu now (I know this is technically not part of your PR but just noticed it)